samples/dnn/models.yml

```diff
@@ -15,6 +15,30 @@ opencv_fd:
   rgb: false
   sample: "object_detection"
 
+# YOLO object detection family from Darknet (https://github.com/AlexeyAB/darknet)
+# Might be used for YOLOv4 and TinyYolov4
+yolov4:
+  model: "yolov4.weights"
+  config: "yolov4.cfg"
+  mean: [0, 0, 0]
+  scale: 0.00392
+  width: 416
+  height: 416
+  rgb: true
+  classes: "object_detection_classes_yolov3.txt"
+  sample: "object_detection"
+
+yolov4-tiny-voc:
+  model: " yolov4-tiny.weights"
```

Comment:
Did you check your code locally?

Reply:
Yes, @alalek i did check, below evidence:

./object_detection --config=yolov4-tiny.cfg --classes=coco.names --model=yolov4-tiny.weights --width=416 --height=416 --scale=0.00392 --input=image-demo.jpg

Thank you for your concern

![image](https://user-images.githubusercontent.com/675645/91115664-12368300-e661-11ea-8089-39cc0a0553f9.png)